### ISSUE TITLE
Alt + Tabbing several times causes the game to crash

### ISSUE THREAD

**Issue by AsciiWolf** [stuntrally/stuntrally#87]
Stunt Rally crashes after Alt + Tabbing multiple times from and to the game with latest Stunt Rally from Git (5ce6dde7273e61bf504e4b60bd288bfeac3a8ecb + https://github.com/stuntrally/tracks/commit/bef80ff5cdec845c3d04b7066c7d96eb025332db) + latest OGRE (13.4.4).

**Comment by cryham** [comment]
Strange. On fullscreen, or windowed?

**Comment by AsciiWolf** [comment]
On fullscreen.

**Comment by AnotherFoxGuy** [comment]
The same happens with RoR, seem to be an OGRE issue

**Comment by cryham** [comment]
I wonder if this has anything to do with plenty of such messages in ogre.log:
Note: Mesh instance 'rd.mesh.15_0' was defined as manually loaded, but no manual loader was provided. This Resource will be lost if it has to be reloaded.
Happens for each road segment. And now I have no idea what that loader would need to do or how to code it if needed.

**Comment by cryham** [comment]
Is this also on Linux?
IIRC on Windows and fullscreen, DirectX could have lost / reset device and unload some textures or meshes?

**Comment by AsciiWolf** [comment]
Yep, also on Linux (GNOME Wayland session with Stunt Rally running on XWayland).
